WolvenKit.Common/RED4/CR2W/JSON/PrimitiveConverter.cs

```diff
@@ -2348,7 +2352,17 @@ public RedClassConverter(ReferenceResolver<RedBaseClass> referenceResolver)
                         {
                             val = JsonSerializer.Deserialize(ref reader, valInfo.Type, options);
                         }
-
+                        /*
+                                                static string? FirstCharToLowerCase( string? str)
+                                                {
+                                                    if (!string.IsNullOrEmpty(str) && char.IsUpper(str[0]))
+                                                        return str.Length == 1 ? char.ToLower(str[0]).ToString() : char.ToLower(str[0]) + str[1..];
+
+                                                    return str;
+                                                }
+                        */
+                        //what's the difference between RedName and Name ?!
```

Comment:
RedName is CDPRs RTTI name, Name is our C# Name (mostly just first char uppercase but there are exceptions)